### ISSUE TITLE
How to calculate confidence score for intent and slots while predicting 

### ISSUE THREAD

**Issue by kowshik226** [monologg/JointBERT#6]
is there a way to calculate prediction/confidence score for intent and slot identification

i try to use intent_logits and slot_logits but i didnt succeed 

can you help me here

Thanks

**Comment by monologg** [comment]
Quite lack of information what you want to do exactly, and the code you made:(

I'm not sure, but instead of using logits, you may use `intent_preds`, `out_intent_label_ids`, `slot_preds_list`, `out_slot_label_lst`.

**Comment by monologg** [comment]
Closing, feel free to reopen if I haven't fully answered your questions.
